pkg/cloudprovider/provider/kubevirt/provider.go

```diff
@@ -261,62 +269,99 @@ func (p *provider) Create(machine *v1alpha1.Machine, _ *cloudprovidertypes.Provi
 		return nil, err
 	}
 
-	quantity, err := resource.ParseQuantity("2Gi")
+	quantity, err := resource.ParseQuantity("10Gi")
 	if err != nil {
 		return nil, err
 	}
 
-	virtualMachineInstance := &kubevirtv1.VirtualMachineInstance{
+	pvcRequest := corev1.ResourceList{
+		corev1.ResourceStorage: quantity,
+	}
+
+	var (
+		vmDataVolume     = fmt.Sprintf("%v-%v", machine.Name, "data-volume")
+		running          = true
+		storageClassName = "kubermatic-fast"
+	)
+	virtualMachine := &kubevirtv1.VirtualMachine{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      machine.Name,
 			Namespace: c.Namespace,
+			Labels: map[string]string{
+				"kubevirt.io/vm": machine.Name,
+			},
 		},
-		Spec: kubevirtv1.VirtualMachineInstanceSpec{
-			Domain: kubevirtv1.DomainSpec{
-				Devices: kubevirtv1.Devices{
-					Disks: []kubevirtv1.Disk{
-						{
-							Name:       "containerdisk",
-							DiskDevice: kubevirtv1.DiskDevice{Disk: &kubevirtv1.DiskTarget{Bus: "virtio"}},
+		Spec: kubevirtv1.VirtualMachineSpec{
+			Running: &running,
```

Comment:
Please use the helpers in `utilpointer "k8s.io/utils/pointer"` for this kind of problem

Reply:
What problem exactly?